### ISSUE TITLE
CI fails because icub-contrib-common is looking for an exact match of the YARP version

### ISSUE THREAD

**Issue by xEnVrE** [xEnVrE/robots-io#4]
The CI keeps failing with this error:

```console
Cloning into 'icub-contrib-common'...
Note: switching to 'v1.19.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at cd63526 added official maintainer
CMake Error at CMakeLists.txt:8 (find_package):
  Could not find a configuration file for package "YARP" that is compatible
  with requested version "3.3.2".

  The following configuration files were considered but not accepted:

    /home/runner/work/robots-io/robots-io/lib/cmake/YARP/YARPConfig.cmake, version: 3.7.2 (64bit)



-- Configuring incomplete, errors occurred!
```

That is strange as `icub-contrib-common` is not searching for an exact match:

https://github.com/robotology/icub-contrib-common/blob/cd6352688c64aadc0a044925532d08fbcb688886/CMakeLists.txt#L8

**Comment by traversaro** [comment]
It is a CMake 3.26.0 regression, fixed in CMake 3.26.1, see https://github.com/robotology/robotology-superbuild/issues/1366 for more details.

**Comment by xEnVrE** [comment]
Thanks @traversaro
